libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.utils.ts

```diff
@@ -36,27 +86,55 @@ export const transformApplicationToParentalLeaveDTO = (
   >(application.answers, 'periods')
   let periods: Period[] = []
 
-  if (periodsAnswer !== null) {
+  if (periodsAnswer) {
     periods = periodsAnswer.map((period) => ({
       from: period.startDate,
-      to: period.endDate,
+      to: period.endDate.split('T')[0],
       ratio: Number(period.ratio),
+      approved: true,
+      paid: false,
     }))
   }
 
-  // 'yes' will not be hard coded once the state machine has been moved into the api module
   const isSelfEmployed =
-    extractAnswer(application.answers, 'employer.isSelfEmployed') === 'yes'
-
-  const employer: Employer = {
-    nationalRegistryId: isSelfEmployed
-      ? application.applicant
-      : extractAnswer(application.answers, 'employer.nationalRegistryId'),
-    email: isSelfEmployed
-      ? extractAnswer(application.answers, 'applicant.email')
-      : extractAnswer(application.answers, 'employer.email'),
+    extractAnswer(application.answers, 'employer.isSelfEmployed') ===
+    BOOLEAN_TRUE
+
+  const employer = getEmployer(application, isSelfEmployed)
+
+  const union: Union = {
+    id: extractAnswer(application.answers, 'payments.union'),
+    name: '',
```

Comment:
`name` is a required attribute for and Union, PensionFund - leaving it empty since we do not save it